src/Slick.hs

```diff
@@ -15,8 +15,6 @@ module Slick
   , makePandocReader'
   , loadUsing
   , loadUsing'
-  , markdownOptions
-  , html5Options
```

Comment:
Curious what the rationale for removing these was; would you mind talking about it?

Reply:
Of course, if they can be customized, they should be exported.

Reply:
To force user provide this options and define them in user space. You think I went too far in forcing this?

Reply:
We don't need to `force` the user to do anything; we want to provide them a smooth path to getting something up and running; if they decide the options aren't right for them they can track down where to change them. Our priority is ease-of-use 😄

Reply:
@ChrisPenner so, what's best course here, restore defaults?

Reply:
Yes, if you prefer we can rename the options to reflect that they're simply "defaults"; e.g. `defaultHTML5Options` or whatever, but since building the options objects is confusing, and annoying I think it's important to provide nice defaults.

Reply:
I like `defaultHTML5Options`.